packages/eslint-plugin-pf-codemods/src/tokenLists/tTokens.ts

```diff
@@ -0,0 +1,856 @@
+// all React tokens in V6, whose value is a Patternfly token variable (has prefix --pf-t)
+// their naming was updated to be prefixed with 't_'
+export const tTokens = [
```

Comment:
Super small nit, but wondering if this would make more sense named "reactTokens" or "tokensPrefixedWithT". Not a blocker and good to merge this as-is, justlet me know what you think.

Reply:
yeah, tokensPrefixedWithT is good. I wouldn't use reactTokens, because also the pf-v6-whatever are react tokens